routes.py

```diff
@@ -0,0 +1,12 @@
+from trytond.wsgi import app
+from trytond.protocols.wrappers import \
+    with_pool_by_alias, with_transaction
+
+
+@app.route('/cryptolog/<database_alias>/callback',
```

Comment:
Il faudrait faire plutôt l'inverse:
```suggestion
@app.route('/<database_alias>/cryptolog/callback',
```